draft-ietf-taps-impl.md

```diff
@@ -723,7 +723,7 @@ Message data.
 In order to parse a received flow of data into Messages, the Message Framer
 notifies the framer implementation whenever new data is available to parse.
 The parameters to the the events and calls for receiving data with a framer
-align with the Receive call in the API ({{Section 9.3 of I-D.ietf-taps-interface}}).
+align with the `Receive` call in the API ({{Section 9.3 of I-D.ietf-taps-interface}}).
```

Comment:
```suggestion
align with the `Receive` action in the API ({{Section 9.3 of I-D.ietf-taps-interface}}).
```

Reply:
Actually, I wonder, do they align with the `Receive` action or with the `Received` (or `ReceivedPartial`) event?

Reply:
I think it does align with the receive action, since it has minimumIncompleteLength, etc.